src/pybind_module/surfio.cpp

```diff
@@ -25,9 +26,9 @@ PYBIND11_MODULE(surfio, m) {
       .def(
           py::init<
               int, double, double, double, double, double, double, int, double, double, double>(),
-          py::arg("nrow"), py::arg("xori"), py::arg("xmax"), py::arg("yori"), py::arg("ymax"),
-          py::arg("xinc"), py::arg("yinc"), py::arg("ncol"), py::arg("rot"), py::arg("xrot"),
-          py::arg("yrot")
+          py::arg("nrow"), py::arg("xori") = 0., py::arg("xmax") = 0., py::arg("yori") = 0.,
```

Comment:
Duplicating default values seems a bit dangerous. Could you add a comment both in the C++ header and here to "if you change defaults, make sure it's changed in the other place"?

Reply:
I can add some comments. I could not find any way for pybind to use the default values of the c++ struct